content/administration/install/install.rst

```diff
@@ -107,7 +107,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 13.0 'deb' package currently supports `Debian Buster`_, `Ubuntu 18.04`_ or above.
+Odoo 15.0 'deb' package currently supports `Debian 11 (Bullseye)`_, `Ubuntu 20.04 (Focal)`_ or above.
```

Comment:
Sorry, I forgot to mention that you also have to change the references accordingly in lines 755 and 759 hence the warnings.